dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -348,3 +352,29 @@ def test_categorical_column_with_wrong_options(self):
                                    "CategoricalColumn parameter 'options' must"
                                    " be of type CategoricalOptions."):
             profiler = CategoricalColumn("Categorical", options="wrong_data_type")
+
+    def test_categorical_column_is_counted(self):
```

Comment:
If we make this function be able to get the top k counts, then we can have this test validate that changing the k value works, however, these should be validated using the existing test, no?